scripts/start.sh

```diff
@@ -60,10 +60,19 @@ fi
 if [ -z "${ARLAS_SERVER_URL}" ]; then
   ARLAS_SERVER_URL="http://demo.arlas.io/arlas/"
   export ARLAS_SERVER_URL
-  echo "The default ARLAS-server url is used"
+  echo "The default ARLAS-server url '${ARLAS_SERVER_URL}' is used"
 else
   echo ${ARLAS_SERVER_URL}  "is used as 'arlas.server.url'"
 fi
+
+if [ -z "${ARLAS_SERVER_COLLECTION}" ]; then
+  ARLAS_SERVER_COLLECTION="demo_ais-danmark"
```

Comment:
In find strange, in an open source project, to reference one of our projects

Reply:
ARLAS-wui needs ARLAS-server to fetch the data

Reply:
Maybe something like `foobar` ou `lorem-ipsum`

Reply:
Sorry, the reference to demo_ais-danmark is strange, not arlas-server